### PR TITLE
RU lang Update andrius.json

### PR DESCRIPTION
**Pull request by Alkmaaar**

### CODE REVIEW

**Comment by Alkmaaar** [comment]
About checks.. Is it okay? If yes i will continue, wish to help you

File: src/data/features/mondstadt/boss/andrius.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
-  "name": { "en": "Andrius (Lupus Boreas)" },
+  "name": { 
+    "en": "Andrius (Lupus Boreas)",
+    "ru": "Андриус (Лупус Бореалис)",
```
Comment (EliteMasterEric) [inline src/data/features/mondstadt/boss/andrius.json]: The comma at the end of this line needs to be removed.

**Review by EliteMasterEric — COMMENTED** [review]
Thanks for taking the time to make the contribution! JSON is very picky sadly, and I haven't gotten around to developing a good system for contributing translations for feature names.

Either you can fix it and I can merge it with a single click, or I can fix it myself once I'm home.

File: src/data/features/mondstadt/boss/andrius.json (hunk 1 above)
Comment (Alkmaaar) [reply src/data/features/mondstadt/boss/andrius.json]: Don't worry about it, not hard to help with this ;) And thank you for remark, i didn't know about this  JSON syntax nuance.

**Pull request merged**